ipatests/test_integration/test_cert.py

```diff
@@ -217,6 +218,135 @@ def test_getcert_list_profile_using_subca(self, test_subca_certs):
                                  "in state {}". format(status))
 
 
+class TestCertmongerRekey(IntegrationTest):
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master, setup_dns=True)
+
+    @pytest.fixture
+    def request_cert(self):
+        """Fixture to request and remove a certificate"""
+        self.request_id = 'test'
```

Comment:
IMO we should randomize the request_id on each request.

Reply:
I think using a fixed request id is fine as long as it is unique to the test (or random). It avoids the requirement to scrape the output of getcert.